website/content/docs/auth/gcp.mdx

```diff
@@ -333,6 +344,8 @@ Read more on the
 
 ### GCE
 
+You can autogenerate this token in Vault versions newer than 1.8.2.
```

Comment:
Vault 1.7.2? - or is this something intended for the future 1.8.2 release :smile:

Reply:
Should be the next release. Who willr review the patch in the plugin review?